src/main/java/com/aws/greengrass/mqtt/bridge/auth/MQTTClientKeyStore.java

```diff
@@ -150,6 +150,14 @@ public synchronized void listenToCAUpdates(UpdateListener listener) {
         updateListeners.add(listener);
     }
 
+    /**
+     * Stop listening to KeyStore updates for a given listener.
+     * @param listener callback
+     */
+    public synchronized void unsubscribeToUpdates(UpdateListener listener) {
```

Comment:
Same as here: https://github.com/aws-greengrass/aws-greengrass-mqtt-bridge/pull/85/files#r1132380603.

You made a good change the the MQTTClient file so that the subscribe won't change, and I think that's sufficient. I don't think we need or actually want to unsubscribe.